packages/bpk-component-navigation-bar/src/BpkNavigationBar.module.scss

```diff
@@ -43,6 +43,10 @@
     tokens.$bpk-surface-default-day
   );
 
+  &--justified {
+    justify-content: space-between;
+  }
+
```

Comment:
For modals where the title is not centred we should justify the title to fill up the whole nav bar.